lib/matterStacks.ts

```diff
@@ -99,6 +99,10 @@ export class MatterStack extends Stack {
                 type: "Number",
                 description: "Validity in days for new PAI(s)"
             }).valueAsNumber;
+            const dacValidityInDays = new CfnParameter(this, "dacValidityInDays", {
```

Comment:
Let's use https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.CfnRuleAssertion.html somewhere below to check that the two validity intervals make sense.

Reply:
Im not sure if this is possible. The only cfn condition we can check is Equals